tracer/src/Datadog.AutoInstrumentation.Generator/Resources/Data/Integration.cs

```diff
@@ -24,7 +24,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.$(Namespace);
     ParameterTypeNames = $(ParameterTypeNames),
     MinimumVersion = "$(MinimumVersion)",
     MaximumVersion = "$(MaximumVersion)",
-    IntegrationName = "$(IntegrationName)")]
+    IntegrationName = $(IntegrationValue)$(IntegrationKind))]
```

Comment:
Renaming `IntegrationName` to `IntegrationValue`, when it's used to set the `IntegrationName` seems an odd choice... but 🤷‍♂️  meh 😄

Reply:
I would personally have kept it as `IntegrationName`, but I repeat, meh 😄 A big fat meh.